README.md

```diff
@@ -115,11 +115,11 @@ For a more information, checkout our [API reference](https://docs.svix.com).
 This project uses [goreleaser](https://github.com/goreleaser/goreleaser/).
  1) Install [go](https://golang.org/doc/install).
  2) Install goreleaser via the steps [here](https://goreleaser.com/install/).
- 3) Build current commit via `goreleaser release --snapshot --skip-publish`.
-
-[release-img]: https://img.shields.io/github/v/release/svixhq/svix-cli
-[release]: https://github.com/svixhq/svix-cli/releases
-[golangci-lint-img]: https://github.com/svixhq/svix-cli/workflows/go-lint/badge.svg
-[golangci-lint]: https://github.com/svixhq/svix-cli/actions?query=workflow%3Ago-lint
-[report-card-img]: https://goreportcard.com/badge/github.com/svixhq/svix-cli
-[report-card]: https://goreportcard.com/report/github.com/svixhq/svix-cli
+ 3) Build current commit via `goreleaser release --snapshot --skip-publish --rm-dist`.
```

Comment:
What's this change?

Reply:
Not _super_ related to this, but I was testing the `goreleaser` changes locally and adding this clears the `dist` folder on run (otherwise the goreleaser build fails).

Reply:
Please let's avoid having unrelated changes in the same PR, or at the very least not in the same commit! :)